src/core/InternalAnimatedValue.js

```diff
@@ -32,6 +32,9 @@ export default class InternalAnimatedValue extends AnimatedNode {
   }
 
   constructor(value, constant = false) {
+    if (value === undefined) {
```

Comment:
maybe?
```suggestion
    if (value == undefined) {
```

Reply:
nope, 0 is okay

Reply:
sure

Reply:
okay, whatever. I think NaN is handled natively in another way, so let's leave it